resources/public/api/conf/2.0/schemas/uk_property_non_fhl_period_summary_create/request.json

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "title": "Create a Historic Non-FHL UK Property Income & Expenses Period Summary",
+  "description": "This endpoint allows a developer to create income, and expenses  for their non-FHL UK property business to make periodic submission for tax years from 2017-18 to 2021-22. A National Insurance number must be provided. The new Property specification endpoints should be used to create periodic submissions for the current tax year.",
+  "type": "object",
+  "properties": {
+    "fromDate": {
+      "description": "The first day that the income and expenses period summary covers.",
+      "type": "string",
+      "example": "2020-01-01"
+    },
+    "toDate": {
+      "description": "The last day that the income and expenses period summary covers.",
+      "type": "string",
+      "example": "2020-01-01"
+    },
+    "income": {
+      "description": "Object containing the details about income",
+      "type": "object",
+      "properties": {
+        "periodAmount": {
+          "description": "Total rents from property, ground rents and rent charges but not rent-a-room.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
```

Comment:
The descriptions of all these properties under income seem to match the spec - but they are mostly all the same so I think the spec must be wrong. It is a question for the BA as to what they should be.